core/templates/pages/preferences-page/form-fields/preferred-languages.component.spec.ts

```diff
@@ -60,25 +60,20 @@ describe('Preferred Languages Component', () => {
       id: 'en',
       text: 'English'
     }];
-    componentInstance.formCtrl = {
-      valueChanges: {
-        subscribe(callb: (val: string) => void) {
-          callb(value);
-        }
-      }
-    } as FormControl;
+    componentInstance.formCtrl = new FormControl(value);
     componentInstance.ngOnInit();
+    fixture.detectChanges();
     expect(componentInstance.chipList.errorState).toBeFalse();
-    value = '';
-    componentInstance.formCtrl = {
-      valueChanges: {
-        subscribe(callb: (val: string) => void) {
-          callb(value);
-        }
-      }
-    } as FormControl;
+    const invalidValue = 'fr';
+    componentInstance.formCtrl.setValue(invalidValue);
```

Comment:
As we are using the `invalidValue` once, should we not directly do
`componentInstance.formCtrl.setValue('fr);`
Similary for the validValue below?

Reply:
I'll make the suggested changes. Thanks!